numba/tests/test_array_exprs.py

```diff
@@ -613,5 +615,21 @@ def do_call(x, y):
         self.assertTrue(isinstance(oty.type.dtype, types.Float))
 
 
-if __name__ == "__main__":
-    unittest.main()
+class TestDatetimeDeltaOps(TestCase):
+    def test_div(self):
+        """
+        Test the division of a timedelta by numeric types
+        """
+        def arr_div(a, b):
+            return a / b
+
+        py_func = arr_div
+        cfunc = njit(arr_div)
+        test_cases = [(np.ones(3, np.dtype(TIMEDELTA_M)), np.ones(3, np.dtype(TIMEDELTA_M))),
+                      (np.ones(3, np.dtype(TIMEDELTA_Y)), np.ones(3, np.dtype(TIMEDELTA_Y))),
+                      (np.ones(3, np.dtype(TIMEDELTA_M)), 1),
+                      (np.ones(3, np.dtype(TIMEDELTA_M)), np.ones(3, np.int64)),
+                      (np.ones(3, np.dtype(TIMEDELTA_M)), np.ones(3, np.float64)),
+                     ]
```

Comment:
Test the case where `a` have dtype `TIMEDELTA_M` and `b` have dtype `TIMEDELTA_Y`:
```python
(np.ones(3, np.dtype(TIMEDELTA_M)), np.ones(3, np.dtype(TIMEDELTA_Y))),`
```

Reply:
thanks, I thought numpy would reject that operation, it actually works. Added two tests.